extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -88,6 +94,15 @@ export class Toolbar {
     } else {
       this.hideToolbar_();
     }
+
+    // Use the placeholder to fill the height of the toolbar
+    this.vsync_.mutate(() => {
+      if (this.placeholder_) {
```

Comment:
The compensating element is within our container that receives display: none; So it would not be shown.

But I can throw this into attempt show before the return.